specification/sql/resource-manager/Microsoft.Sql/preview/2024-08-01-preview/ServerAdvisors.json

```diff
@@ -583,7 +583,9 @@
         "details": {
           "description": "Gets additional details specific to this recommended action.",
           "type": "object",
-          "additionalProperties": true,
+          "additionalProperties": {
+            "type": "string"
+          },
```

Comment:
how about this one? 

https://github.com/Azure/azure-rest-api-specs/blob/fc375a33a4386ee0a0935b66c6055ffc1a6dc630/specification/sql/resource-manager/Microsoft.Sql/preview/2024-08-01-preview/DatabaseAdvisors.json#L409

Reply:
@UrosKrsticMIPerf can you help to take a look at this one?

Reply:
Will do.

Reply:
Added as well and it is updated, no longer "additionalProperties": true